_posts/2022-02-02-#748.md

```diff
@@ -10,19 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- [libsecp256k1](https://github.com/bitcoin-core/secp256k1/) is the library that Bitcoin Core uses for low-level cryptographic operations such as signing and verification.
+
+- This PR adds usage examples for several common use cases: ECDSA signatures, Schnorr signatures, and (elliptic curve) Diffie-Hellman key exchange. There are no existing usage examples in the library.
+
+- The examples are intended to clarify the API by supplementing the documentation in the [various header files](https://github.com/bitcoin-core/secp256k1/tree/master/include).
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+2. Can you compile and run the example (note: you must run `./configure` with the `--enable-examples` flag)?
+3. Why do the examples demonstrate how to obtain randomness? Is this a good idea?
+4. Are those the correct ways to obtain randomness on each of the supported operating systems?
+5. Can you follow the examples?
```

Comment:
Potential follow-up: "Can you follow the examples? Is it clear how they should be generalized to a production system?"

Reply:
Good idea. Added.